### ISSUE TITLE
Add support for zkSync 2.0 to the crypto-batch-payment tool

### ISSUE THREAD

**Issue by iglesiasbrandon** [storj/crypto-batch-payment#1]
## Describe the feature or problem you'd like to solve
crypto batch payment tool only supports zkSync 1.0 at the moment; 2.0 is getting released in the next 30-90 days, so we need to add support for 2.0 into the tool so that we can start to utilize it when it's in production for our node operator payments.

## Acceptance Criteria
- we can make zksync 2.0 payments on test net
- make the paymaster address configurable

## Notes
- https://github.com/matter-labs/zksync-era/blob/f5a36684b8c2088d36308bad86643bc7ab3d6a78/core/lib/types/src/transaction_request.rs#L522

**Comment by iglesiasbrandon** [comment]
From @elek, currently blocked by https://github.com/zksync-sdk/zksync2-go/issues/3

**Comment by elek** [comment]
https://github.com/storj/crypto-batch-payment/pull/2

**Comment by shaupt131** [comment]
Do we still need a follow-up ticket to check the number of confirmations after a transaction?

**Comment by elek** [comment]
> Do we still need a follow-up ticket to check the number of confirmations after a transaction?

Yep. But it's not ZkSync specific (we neither do any check for Ethereum). And based on my understanding it's not so urgent...
